grunt-configs/webfontjson.js

```diff
@@ -584,6 +586,6 @@ module.exports = function(grunt, options) {
                     }
                 ]
             }
-        }
+        },
     };
 };
```

Comment:
odd changes

you'll also need to create json files for the `auto` hinted fonts